CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # grommet-theme-hpe
 
+## 6.4.0
+
+### Minor Changes
+
+- f3702f9: - Updated Tab styles to use underline marker instead of background fill to indicate the selected tab. This removes potential conflict in visual hierarchy with primary buttons.
```

Comment:
```suggestion
- f3702f9: - Updated Tab styles to use underline marker instead of background fill to indicate the selected tab. This removes potential conflict in visual hierarchy with primary Buttons.
```